### DISCUSSION TITLE
Starting a new project

### DISCUSSION THREAD

**Discussion by karlguillotte** [contember/admin#712] [Support] [answered]
I am working on a proof of concept using Contember at work. 

I ran the installation from the documentation website, https://docs.contember.com/intro/installation. It seems that all the dependencies are out of date, the created project does not include the new @contember/layout dependencies as suggest in the documentation. 

So my question is, am I missing something or the project creation template should be updated ? Should I be using the examples to scaffold a new project ? 

Many thanks!

**Comment by jonasnobile** [comment, marked as answer]
Hello! Thank you so much for your interest in our framework. We've got lots of exciting new things in the works right now. Unfortunately, these new things come with a few changes here and there, so old versions might not be compatible.

Our new projects start in [Contember AI Studio](https://www.contember.com/studio). Studio creates projects from scratch with a new template. I've uploaded the new template to this repo [template](https://github.com/contember/examples/tree/next/template) so you can take a look at it publicly. You can copy the template to use it as start for your new project.

**Reply by jonasnobile** [reply]
Let me know, If you need any other support.

**Reply by karlguillotte** [reply]
Fabulous, thanks for your reply @jonasnobile ! 

I will look have a look at the AI and existing templates. With me question and your answer, someone else having similar questions to mine will have useful insights.
